### PR TITLE
Quality of life improvements

### PR DESCRIPTION
**Pull request by pitr**
Tooltips on how alias variables works
Screen Shot 2019-07-17 at 14 57 46

Sort zmon.check.... higher than other checks
Screen Shot 2019-07-17 at 14 58 43

Link to zmon check
Screen Shot 2019-07-17 at 16 15 21

### CODE REVIEW

File: dist/directives/metric_name_field.ts [hunk 1]
```diff
@@ -45,6 +53,17 @@ export class MetricNameFieldCtrl {
         }
         this.aliasInputVisible = false;
     }
+
+    private sortForZmon(left, right): number {
+        // prioritize metric names that start with z
```
Comment (alexkorotkikh) [inline dist/directives/metric_name_field.ts:58]: 💚

**Comment by alexkorotkikh** [comment]
why all the changes are doubled? Do we really need to have `dist` dir under version control?

**Comment by lfroment0** [comment]
:+1:

**Comment by pitr** [comment]
alternatively `dist` folder can be generated every time by grafana-deploy build. I chose this way, since parent (OSS) repo has a dist folder

**Comment by pitr** [comment]
👍

**Pull request merged**